lib/Deserializer/functions.php

```diff
@@ -298,6 +298,8 @@ function mixedContent(Reader $reader): array
  * You can use, e.g., a named constructor (factory method) to create an object using
  * this function.
  *
+ * @param callable $func
```

Comment:
Do we really need this, since it does not provide additional value?

Reply:
to make phpstan happy

Reply:
FYI: I moved the level down to 6 and removed all these changes - I was anyhow not that convinced ....

Reply:
Thx. Agree on this.